apps/api/src/reservation/api/http/inventory.controller.ts

```diff
@@ -71,10 +76,12 @@ export class InventoryController {
     @Query('startDate') startDate: string,
     @Query('endDate') endDate: string,
   ) {
-    return this.inventoryService.getAll({
-      pagination: req.pagination,
-      startDate: startDate && new Date(startDate),
-      endDate: endDate && new Date(endDate),
-    });
+    return this.queryBus.execute(
+      new GetInventoryItemsQuery(
+        startDate ? new Date(startDate) : undefined,
+        endDate ? new Date(endDate) : undefined,
```

Comment:
If `startDate` and `endDate` are undefined if they are not specified, can we just write `startDate: startDate && new Date(startDate)`